packages/widgets/src/splitpanel.ts

```diff
@@ -341,6 +350,9 @@ export class SplitPanel extends Panel {
     this._pressData.override.dispose();
     this._pressData = null;
 
+    // Emit the handle moved signal.
+    this._handleMoved.emit(undefined);
```

Comment:
You could drop the `undefined`, isn't it?

```suggestion
    this._handleMoved.emit();
```